packages/core/package.json

```diff
@@ -19,6 +19,7 @@
     "@types/ws": "^3.0.2",
     "@types/yargs": "^8.0.2",
     "ajv": "^5.2.2",
+    "axios": "^0.18.0",
```

Comment:
why don't use standard APIs, like `fetch`?

Reply:
I forgot about that, nice idea !

Reply:
Looking into it, I was relying a bit on timeouts so that the request doesn't take too long, `fetch` doesn't seem to provide timeouts.

I could simply use `Promise.race` or something similar, but this could mean that the connection is still happening despite the race rejection by the timeout.

Reply:
ok, just want to avoid unnecessary dependencies in core